app/src/lib/stores/app-store.ts

```diff
@@ -351,6 +351,8 @@ const hideWhitespaceInChangesDiffDefault = false
 const hideWhitespaceInChangesDiffKey = 'hide-whitespace-in-changes-diff'
 const hideWhitespaceInHistoryDiffDefault = false
 const hideWhitespaceInHistoryDiffKey = 'hide-whitespace-in-diff'
+const hideWhitespaceInPullRequestDiffDefault = false
+const hideWhitespaceInPullRequestDiffKey = 'hide-whitespace-in-diff'
```

Comment:
🤦

Reply:
Random thoughts.. It would be totally cool if after accepting suggestions, I could press a button (or automatically run) on a PR to run some prettifier setup for the repo so I don't have to bounce back to local dev environment to run it. :D